### ISSUE TITLE
Why normalizing HR images to [-1, 1] whereas normalizing LR images to [0, 1]?

### ISSUE THREAD

**Issue by DreamChaserMXF** [thunil/TecoGAN#14]
Hi, I was wondering why you normalize HR images to [-1, 1] whereas LR images were normalized to [0, 1] ?

https://github.com/thunil/TecoGAN/blob/027867b7f81b2b2fc463875cb5626e8cd801f870/lib/dataloader.py#L318

https://github.com/thunil/TecoGAN/blob/027867b7f81b2b2fc463875cb5626e8cd801f870/lib/dataloader.py#L324

**Comment by ManuelCarullo** [comment]
Sorry for bringing this up after such a long time, but did you ever figure out why this is the case? I tried to change normalizing LR and HR images to [-1,1] in my own implementation to maybe improve performance, but I couldn't figure out the additional changes needed to the various preprocess and deprocess calls to get consistent results. 

Right now it seems like I'm having issues with this line. https://github.com/thunil/TecoGAN/blob/3a75d691f7b5bd3482ecfb2d731d6ea6099c619f/lib/Teco.py#L143
I'm thinking I can remove the preprocess/deprocess calls here, if both LR and HR footage is normalized to [-1,1], but I get weird color shifts/ flickering in the WarpPreGen tensorboard summaries, when I do. 

I'm using my own paired LR data, not the default gaussian downsample from HR training data, if that changes anything.

**Comment by DreamChaserMXF** [comment]
It seems that two different normalizing strategies originate from historic reason and don't have huge impact to the performance.
The color shifts/flickering may be caused by something else, in my opinion.
